src/mainsection.jsx

```diff
@@ -0,0 +1,14 @@
+export default function MainSection() {
+    return (
+        <>
```

Comment:
Change this fragment to the `<section>` below. Everything needs to be wrapped in a container and since you're already doing so below you can just use the section.